### ISSUE TITLE
Only first 10 team drives are listed

### ISSUE THREAD

**Issue by danjjl** [jupyterlab/jupyterlab-google-drive#116]
If a user has more than 10 team drives, only the first ten are listed.

This is a bug in `func listTeamDrives` ([src/drive/drive.ts#L1113](https://github.com/jupyterlab/jupyterlab-google-drive/blob/master/src/drive/drive.ts#L1113))

By default `teamdrives.list` returns a maximum of 10 teams per page (see [documentation](https://developers.google.com/drive/v3/reference/teamdrives/list)).

To fix this bug the `nextPageToken` value should be queried and used to fetch the remaining pages (see example [here](https://developers.google.com/drive/v3/web/manage-teamdrives)).

*(I would have submitted a PR for this myself but my js level is close to NULL)*

**Comment by ian-r-rose** [comment]
Hi @danjjl, thanks for the report! You are completely correct. I should be able to put together fix for this sometime in the next week. I personally don't have access to many team drives, so some user testing may be helpful :)

**Comment by ian-r-rose** [comment]
@danjjl If you are so inclined, would you be able to test if #118 fixes the issue in the case of >10 team drives? You would need to check out the `depaginate` branch of `@jupyterlab/google-drive`, then in that directory run
```bash
jlpm install
jlpm run build`
jupyter labextension install .
```
in order to get a local install up and running.

**Comment by danjjl** [comment]
Well done!

It now works fine.

*(with the little "bug" that if two team drives have the same name, jupyterlab lists both of them but can only enter the first one - This is the same bug as https://github.com/jupyterlab/jupyterlab-google-drive/issues/96 and should not be a problem for most users (I only came across it while creating extra fake team drives))*

**Comment by ian-r-rose** [comment]
Great, thanks for checking!

Yeah, #96 is a real thorn in my side, but I don't really see any way around it at the moment -- there is a fundamental difference between the path-based API that the notebook server expects and the uuid-based API that Google Drive uses that is really tough to paper over.

**Comment by ian-r-rose** [comment]
@danjjl I have just published `v0.10.1`, which includes this fix.
